### DISCUSSION TITLE
Sorting by column breaks without informative message

### DISCUSSION THREAD

**Discussion by tinoptilogic** [RevealBi/Reveal.Sdk#117] [General]
https://www.screencast.com/t/UyhA4TT6Zby

Trying to sort on a column in the FacilitySummary table is leading to an unknown error - using a calculated field to accomplish the same sort works.

**Comment by TorreyBetts** [comment]
Based on the error, it sounds like it's related to your data source processing data on the server. Is that intended? Is it possible for you to disable that?

**Reply by tinoptilogic** [reply]
Will investigate and let you know if we find anything. Thanks for help.
